### DISCUSSION TITLE
Am doing poc and followed the OPA tutorial : 3 errors occurred during loading: example.rego:3: rego_parse_error: rule name conflicts with built-in function example.rego:4: rego_parse_error: multiple expressions cannot be used for rule head service-opa.yaml: merge error

### DISCUSSION THREAD

**Discussion by gviswalingam** [open-policy-agent/community#422] [OPA and Rego] [answered]
Followed this document https://www.openpolicyagent.org/docs/latest/deployments/

Code is below 

apiVersion: apps/v1
kind: Deployment
metadata:
  name: opa
  labels:
    app: opa
spec:
  replicas: 1
  selector:
    matchLabels:
      app: opa
  template:
    metadata:
      labels:
        app: opa
      name: opa
    spec:
      containers:
      - name: opa
        image: openpolicyagent/opa:0.52.0
        ports:
        - name: http
          containerPort: 8181
        args:
        - "run"
        - "--ignore=.*"  # exclude hidden dirs created by Kubernetes
        - "--server"
        - "/policies"
        volumeMounts:
        - readOnly: true
          mountPath: /policies
          name: example-policy
      volumes:
      - name: example-policy
        configMap:
          name: example-policy

kubectl create -f deployment-opa.yaml

got a error as below 

3 errors occurred during loading:
example.rego:3: rego_parse_error: rule name conflicts with built-in function
example.rego:4: rego_parse_error: multiple expressions cannot be used for rule head
service-opa.yaml: merge error

Can you please suggest me a fix ?

Thanks

**Comment by anderseknert** [comment]
Hi there! Did you make any modifications to the example.rego file? There's no built-in function called `greeting`, so I'm not sure why you'd see an error for that.

**Reply by gviswalingam** [reply, marked as answer]
Its resolved , it was just Indentation problem. fixed now Thanks
